build/update-attribution-files/create_pr.sh

```diff
@@ -118,6 +118,19 @@ EOF
     pr:create "$pr_title" "$commit_message" "$pr_branch" "$pr_body"
 }
 
+function pr::create::go-mod() {
```

Comment:
This is outside the scope of this PR, but it'd be nice if we automatically added the GitHub labels for the versions (like `v1.22`) that were changed. It's helpful for when I'm doing the changelogs for new releases.

Reply:
Just made an issue https://github.com/aws/eks-distro/issues/972